modules/backend/calendar/WorkUnit.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "../clinic/room/iRoom.hpp"
+#include "../clinic/staff/doctors/iDoctor.hpp"
+#include "TimeSlot.hpp"
+
+namespace calendar
+{
+
+class WorkUnit
+{
+  public:
+    explicit WorkUnit(clinic::staff::doctor::iDoctor* doc, clinic::room::iRoom* room);
+    void addNewTimeSlot(std::unique_ptr<calendar::TimeSlot> timeSlot);
+    clinic::staff::doctor::iDoctor* getDoctor() const;
+    clinic::room::iRoom* getRoom() const;
+
+  private:
+    std::vector<std::unique_ptr<calendar::TimeSlot>> _slot_collection;
```

Comment:
Follow with underscore at the end
Instead collection I'd rather see time_slots (multiple of timeslot)

Reply:
done